src/profile/contact-info/ContactInfoForm.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import {
+  TextField, InputLabel, Button, ButtonGroup,
+} from '@material-ui/core';
+import Select from 'react-select';
+import { useQuery } from '@apollo/client';
+import { GET_COUNTRIES } from '../../queries/CountryQueries';
+import Info from '../../Info';
+
+function ContactInfoForm({ contactInfo: actualContactInfo, onSave, onCancel }) {
+  const { data, loading, error } = useQuery(GET_COUNTRIES);
+  const [contactInfo, setContactInfo] = useState(actualContactInfo);
+
+  const memoCountries = React.useMemo(() => (
+    data?.counties.map((d) => ({ value: d, label: d.name }))
+  ), [data]);
+
+  const onSaveClick = () => {
+    onSave(contactInfo);
+    onCancel();
+  };
+
+  return (
+    <>
+      <Info loading={loading} error={!!error} />
+      <form>
+        <TextField
+          fullWidth
+          value={contactInfo?.email || ''}
```

Comment:
Do you really need to provide fallback values for each TextField if the value is undefined?

Reply:
If I'm not providing any fallback value for each TextField the console will yell at me
```Warning: A component is changing an uncontrolled input to be controlled. This is likely caused by the value changing from undefined to a defined value, which should not happen. Decide between using a controlled or uncontrolled input element for the lifetime of the component. More info: https://reactjs.org/link/controlled-components```